cloudigrade/account/util.py

```diff
@@ -0,0 +1,47 @@
+"""Various utility functions for the account app."""
+import collections
```

Comment:
I'm seeing some inconsistency with this. Sometimes there is a blank space after the doc string, sometimes there isn't. We should settle on one (as I don't think its defined in the PEP) and have it be uniform through our repo.

Reply:
At a quick glance, I didn't see an option to make flake8 opinionated about spacing after the module doc, and pyflakes (which is normally _much_ more opinionated about _many_ more things) also didn't report on it either way. So, for now I just manually removed the spaces since that appears to be our prevailing convention.